doc/manual/src/contributing/testing.md

```diff
@@ -10,6 +10,13 @@ under `src/{library_name}/tests` using the
 You can run the whole testsuite with `make check`, or the tests for a specific component with `make libfoo-tests_RUN`.
 Finer-grained filtering is also possible using the [--gtest_filter](https://google.github.io/googletest/advanced.html#running-a-subset-of-the-tests) command-line option, or the `GTEST_FILTER` environment variable.
 
+### Characterization testing
+
+See below for a broder discussion.
```

Comment:
below?

Reply:
There was an existing long section on characterization testing, but within the functional tests. I am not sure what to put here, but I figured I shouldn't just repeat the same information. Certainly could be better! (e.g. maybe since this comes first most of the information should be moved here, and the other section should reference it.)